pkg/browsers/console.go

```diff
@@ -50,6 +50,12 @@ var ServiceMap = map[string]string{
 	"iam":            "iamv2",
 }
 
+var globalServiceMap = map[string]string{
+	"iam":     "iam",
+	"route53": "route53",
+	"r53":     "r53",
+}
```

Comment:
If we don't need to lookup anything this might be better as `map[string]bool` as it was a bit confusing whether these are meant to indicate particular URLs in AWS for global services. The `r53` doesn't actually point to `r53.amazonaws.com` as an example.

```suggestion
var globalServiceMap = map[string]bool{
	"iam":     true,
	"route53": true,
	"r53":     true,
}
```